test/asset.ts

```diff
@@ -0,0 +1,203 @@
+import {assert} from 'chai'
+import {
+    Asset as AntelopeAsset,
+    APIClient,
+    FetchProvider,
+    Int64,
+    Name,
+    Serializer,
+} from '@wharfkit/antelope'
+import {Chains} from '@wharfkit/common'
+import {mockFetch} from '@wharfkit/mock-data'
+import {PlaceholderAuth} from '@wharfkit/signing-request'
+import type {Asset} from '$lib'
+import {
+    AtomicAssetsAPIClient,
+    AtomicAssetsContract,
+    AtomicAssetsKit,
+    AtomicUtility,
+    Types,
+} from '$lib'
+
+const client = new APIClient({
+    provider: new FetchProvider(Chains.WAX.url, {fetch: mockFetch}),
+})
+
+// Setup the API
+const atomicassets = new AtomicAssetsAPIClient(
+    new APIClient({
+        provider: new FetchProvider('https://wax.api.atomicassets.io/', {fetch: mockFetch}),
+    })
+)
+
+const utility = new AtomicUtility('https://wax.api.atomicassets.io/', Chains.WAX, {
+    client,
+    atomicClient: atomicassets,
+})
+
+const kitInst = new AtomicAssetsKit(utility)
```

Comment:
No, here 3 things are used because we need to mock fetch. For normal usage, users only need to make AtomicUtility and AtomicAssetsKit. AtomicUtility only need url and chain params, other params are optional.